commands/init.go

```diff
@@ -205,7 +199,14 @@ func setIPInConfigs() (err error) {
 		}
 	}
 
+	names := ""
+	for _, cfg := range configs {
+		names += fmt.Sprintf("- %s\n", cfg.name)
+	}
+
 	log.Info("✅  IP Address updated!\n\n")
```

Comment:
Can we print ip address for end user here?

Reply:
It's elsewhere, Ill move it out for more clarity